server/apps/main/forms.py

```diff
@@ -50,6 +50,16 @@ class ShareExperienceForm(forms.Form):
     research = forms.BooleanField(label = "Share for research", required=False)
     research.group = 3
 
+    # authorship option
+    authorship = [
```

Comment:
So here something like 
`authorship = [("Someone else", "Yes"), ("Myself", "No")]`

 `authorship_relation = forms.ChoiceField(widget=forms.RadioSelect, choices=authorship, label="")`

could work - there will be two options to the question _Is this an experience shared on behalf of someone else?_ which would be saved as "Someone else" or "Myself" in the form data. 

Then have a free-text box that is disabled unless "Someone else" is clicked - could do a javascript _onchange_ to manage that